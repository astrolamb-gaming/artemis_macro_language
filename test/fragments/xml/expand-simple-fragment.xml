<mission_data version="1.0">
    <templates>
        <template name="hello">
            <big_message title="${message}" />
        </template>
    </templates>

    <start>
        <expand template="hello" message="Hello, World" />
    </start>
</mission_data>