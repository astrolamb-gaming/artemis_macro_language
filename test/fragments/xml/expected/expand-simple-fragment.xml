<mission_data version="1.0">
    <start>
        <big_message title="Hello, World"/>
        <big_message title="No skip"/>
        <big_message title="False skip"/>
        <big_message title="False macro skip"/>
    </start>
</mission_data>